library/std/src/os/unix/fs.rs

```diff
@@ -1040,6 +1040,7 @@ pub fn fchown<F: AsFd>(fd: F, uid: Option<u32>, gid: Option<u32>) -> io::Result<
 /// }
 /// ```
 #[stable(feature = "unix_chown", since = "1.73.0")]
+#[cfg(not(any(target_os = "vxworks")))]
```

Comment:
Sorry, this is public stable API. It's reachable by callers of std. We can't cfg it away for VxWorks as long as it claims to be a "Unix".

Instead, add an implementation of `sys::fs::lchown` for VxWorks that just returns a not-supported error always.